docs/src/explanation/supplemental_attributes.md

```diff
@@ -15,11 +15,75 @@ flowchart LR
 
 Supplemental attributes can also contain timeseries in the same that a component can allowing the user to model time varying attributes like outage time series or weather dependent probabilities. See the section [`Working with Time Series Data`](@ref tutorial_time_series) for details on time series handling.
 
-## Getting the attributes in a component or system
+## Getting the attributes in a system
 
-You can retrieve the attributes in a component using the function [`get_supplemental_attributes`](@ref)
+You can retrieve the attributes in a system using the function [`get_supplemental_attributes`](@ref).
+You must pass a supplemental attribute type, which can be concrete or abstract. If you pass an abstract type, all concrete types
+that are subtypes of the abstract type will be returned.
 
-## Getting the components in an attribute
+```julia
+for outage in get_supplemental_attributes(FixedForcedOutage, system)
+    @show summary(outage)
+end
+```
+
+You can optionally pass a filter function to reduce the returned attributes. This example will
+return only FixedForcedOutage instances that have a mean time to recovery greater than or equal to 0.5.
+
+```julia
+for outage in get_supplemental_attributes(
+    x -> get_mean_time_to_recovery(x) >= 0.5,
+    FixedForcedOutage,
+    system,
+)
+    @show summary(outage)
+end
+```
+
+## Getting the attributes associated with a component
```

Comment:
We also have the inverse of this method: `get_associated_components` and `get_associated_available_components`. Should we list it here? I'm happy to add it. I'm concerned about having too many options.

Reply:
I think we need to list both. But that isn't blocking